packages/hooks/src/__tests__/useDepositFiatAddress.spec.tsx

```diff
@@ -30,7 +30,7 @@ describe('useDepositFiatAddress', () => {
 
         result.current.resend();
 
-        expect(result.current.data).toBe('https://example.com&DarkMode=on');
+        expect(result.current.data).toBe('https://example.com/?DarkMode=on');
```

Comment:
why do we have these changes in this PR?

Reply:
@amina-deriv This is the changes needed for cashier side, that time we merge is merge into feature branch 🙏